blob/s3blob/s3blob.go

```diff
@@ -25,6 +25,9 @@
 // https://docs.aws.amazon.com/sdk-for-go/api/aws/session/.
 // The following query options are supported:
 //  - region: The AWS region for requests; sets aws.Config.Region.
+//  - endpoint: The endpoint URL (hostname only or fully qualified URI); sets aws.Config.Endpoint.
+//  - disableSSL: Disable SSL when sending requests; sets aws.Config.DisableSSL.
```

Comment:
Can you clarify that only the value of "true" has this effect?

E.g., something like 

```
//  - disableSSL: A value of "true" disables SSL when sending requests; sets ....

```